spec/till_spec.rb

```diff
@@ -0,0 +1,53 @@
+describe 'Till' do
+  let(:till){Till.new Receipt}
+
+  context 'created with' do
+    it 'a total of 0' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.